packages/pn-pa-webapp/src/navigation/routes.tsx

```diff
@@ -19,6 +20,7 @@ function Router() {
           <Route path={routes.DASHBOARD} element={<Dashboard />} />
           <Route path={routes.DETTAGLIO_NOTIFICA} element={<NotificationDetail />} />
           <Route path={routes.NUOVA_NOTIFICA} element={<NewNotification />} />
+          <Route path={routes.STATISTICHE} element={<Statistics />} />
```

Comment:
Anche se la voce di menù è nascosta, se l'utente ha ruolo operatore accede comunque alla pagina delle statistiche inserendo la url direttamente nella barra del browser.
come soluzione si potrebbe definire la rotta delle statistiche in questo modo
`<Route path={routes.STATISTICHE} element={<RouteGuard roles={[PNRole.ADMIN]} />}>
            <Route path={routes.STATISTICHE} element={<Statistics/>} />
          </Route>`
allo stesso livello di
`<Route path="/"  element={<RouteGuard roles={[PNRole.ADMIN, PNRole.OPERATOR]} />}>`